cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,138 @@
+  - [Cloud Organization audit log export](cloud-org-audit-logs.html), which captures audit logs when many types of events occur in your {{ site.data.products.db }} organization, such as when a cluster is created or when a user is added to or removed from an organization.
+  - Your cloud provider's [IAM roles](../stable/cloud-storage-authentication.html), which provide an extra layer of authentication when your cluster or {{ site.data.products.db }} write data to your cloud provider, such as managed backups or [change data capture (CDC)](../stable/change-data-capture-overview.html) operations.
+
+In addition, follow these guidelines to ensure that your cluster is compliant with PCI DSS 3.2.1:
+
+- Before you insert cardholder data into the cluster, you must first protect it by a combination of encryption, hashing, masking, and truncation. For an example implementation, refer to [Integrate {{ site.data.products.dedicated }} with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html).
+- The cryptographic materials used to protect cardholder data must themselves be protected at rest and in transit, and access to the unencrypted key materials must be strictly limited only to approved individuals.
+- Within the cluster, you must restrict access to cardholder data on a "need to know basis" basis. Access to tables and views in the cluster that contain cardholder data must be restricted, and you must regularly test for compliance. Refer to [Authorization](/docs/{{site.versions["stable"]}}/authorization.html).
+- You must protect networks that transmit cardholder data from malicious access over the public internet, and you must regularly test for compliance. For more information about protecting the cluster's networks, refer to [Network Authorization](network-authorization.html).
+- You must regularly test all systems, applications, and dependencies (including, but not limited to, application libraries, runtimes, container images, and testing frameworks) that are involved in storing or processing cardholder data for known vulnerabilities, you must regularly apply updates, and you must regularly test for compliance. Important security and stability updates are applied regularly and automatically to {{ site.data.products.dedicated }} clusters. These updates include, but are not limited to, the cluster's CockroachDB runtime, the operating systems of cluster nodes, APIs, and management utilities. You are notified about upcoming cluster maintenance before it happens, when it starts, and when it completes. To configure notifications in the {{ site.data.products.db }} Console, refer to [Monitoring and Alerting](/docs/stable/monitoring-and-alerting.html). You can also read the release notes for [CockroachDB](/docs/releases/index.html) and for [{{ site.data.products.db }}](/docs/releases/cloud.html).
```

Comment:
This should be removed. We are taking care of all of this, we shouldn't be telling customers to do so.

Reply:
I wrote this below.